### PR TITLE
Convert default/scripting/species/SP_ABADDONI.focs

### PR DESCRIPTION
**Pull request by o01eg**
Follows up #4161

Dump difference:
```diff
@@ -413,7 +413,7 @@
             accountinglabel = "GREAT_INFLUENCE_LABEL"
             priority = 105
             effects =
-                SetTargetInfluence value = Value + (1.500000 - 1.000000) * abs(Value)
+                SetTargetInfluence value = Value + 0.500000 * abs(Value)
         EffectsGroup // from SP_ABADDONI
             scope =
                 Source
@@ -487,7 +487,7 @@
             accountinglabel = "UNINHABTIABLE_ENVIRONMENT_LABEL"
             priority = 30
             effects =
-                SetTargetPopulation value = -(999.000000)
+                SetTargetPopulation value = -999.000000
         EffectsGroup // from SP_ABADDONI
             scope =
                 Source
@@ -670,7 +670,7 @@
                     Planet
                     Not
                         Focus name = "FOCUS_INDUSTRY"
-                    (0 = LocalCandidate.TurnsSinceFocusChange)
+                    (LocalCandidate.TurnsSinceFocusChange = 0)
                 ]
             priority = 600
             effects =
@@ -683,7 +683,7 @@
                     Planet
                     Not
                         Focus name = "FOCUS_RESEARCH"
-                    (0 = LocalCandidate.TurnsSinceFocusChange)
+                    (LocalCandidate.TurnsSinceFocusChange = 0)
                 ]
             priority = 600
             effects =
@@ -696,7 +696,7 @@
                     Planet
                     Not
                         Focus name = "FOCUS_INFLUENCE"
-                    (0 = LocalCandidate.TurnsSinceFocusChange)
+                    (LocalCandidate.TurnsSinceFocusChange = 0)
                 ]
             priority = 600
             effects =
@@ -709,7 +709,7 @@
                     Planet
                     Not
                         Focus name = "FOCUS_STOCKPILE"
-                    (0 = LocalCandidate.TurnsSinceFocusChange)
+                    (LocalCandidate.TurnsSinceFocusChange = 0)
                 ]
             priority = 600
             effects =
@@ -1000,7 +1000,7 @@
                 ]
             priority = 500
             effects =
-                SetConstruction value = Value + min(max(Value(Target.TargetConstruction) - Value, -(1.000000)), 1.000000)
+                SetConstruction value = Value + min(max(Value(Target.TargetConstruction) - Value, -1.000000), 1.000000)
         EffectsGroup // from SP_ABADDONI
             scope =
                 Source
@@ -1019,9 +1019,9 @@
                 ]
             priority = 500
             effects = [
-                SetResearch value = Value + min(1.000000, max(0.000000, Value(Target.Happiness)) / 5.000000, max(min(-(1.000000), (Value(Target.TargetResearch) - Value) / 5.000000), Value(Target.TargetResearch) - Value))
-                SetIndustry value = Value + min(1.000000, max(0.000000, Value(Target.Happiness)) / 5.000000, max(min(-(1.000000), (Value(Target.TargetIndustry) - Value) / 5.000000), Value(Target.TargetIndustry) - Value))
-                SetInfluence value = Value + min(1.000000, max(min(-(1.000000), (Value(Target.TargetInfluence) - Value) / 5.000000), Value(Target.TargetInfluence) - Value))
+                SetResearch value = Value + min(1.000000, max(0.000000, Value(Target.Happiness)) / 5.000000, max(min(-1.000000, (Value(Target.TargetResearch) - Value) / 5.000000), Value(Target.TargetResearch) - Value))
+                SetIndustry value = Value + min(1.000000, max(0.000000, Value(Target.Happiness)) / 5.000000, max(min(-1.000000, (Value(Target.TargetIndustry) - Value) / 5.000000), Value(Target.TargetIndustry) - Value))
+                SetInfluence value = Value + min(1.000000, max(min(-1.000000, (Value(Target.TargetInfluence) - Value) / 5.000000), Value(Target.TargetInfluence) - Value))
             ]
         EffectsGroup // from SP_ABADDONI
             scope =
@@ -1030,9 +1030,9 @@
                 Ship
             priority = 500
             effects = [
-                SetResearch value = Value + min(max(Value(Target.TargetResearch) - Value, -(1.000000)), 1.000000)
-                SetIndustry value = Value + min(max(Value(Target.TargetIndustry) - Value, -(1.000000)), 1.000000)
-                SetInfluence value = Value + min(max(Value(Target.TargetInfluence) - Value, -(1.000000)), 1.000000)
+                SetResearch value = Value + min(max(Value(Target.TargetResearch) - Value, -1.000000), 1.000000)
+                SetIndustry value = Value + min(max(Value(Target.TargetIndustry) - Value, -1.000000), 1.000000)
+                SetInfluence value = Value + min(max(Value(Target.TargetInfluence) - Value, -1.000000), 1.000000)
             ]
         EffectsGroup // from SP_ABADDONI
             scope =
@@ -1360,7 +1360,7 @@
             accountinglabel = "CAPITAL_POOR_CONNECTION_LABEL"
             priority = 100
             effects =
-                SetTargetHappiness value = Value + max(-(5.000000), 5.000000 - Statistic Min value = (JumpsBetween) // StaticCast{i,d}
+                SetTargetHappiness value = Value + max(-5.000000, 5.000000 - Statistic Min value = (JumpsBetween) // StaticCast{i,d}
      condition = And [
     Or [
         Capital
```

### CODE REVIEW

File: default/scripting/species/SP_ABADDONI.focs.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    gameplay_description="SP_ABADDONI_GAMEPLAY_DESC",
+    playable=True,
+    can_produce_ships=True,
+    can_colonize=True,
+    tags=[
+        "LITHIC",
+        "BAD_RESEARCH",
+        "GREAT_INFLUENCE",
+        "GOOD_HAPPINESS",
+        "AVERAGE_SUPPLY",
```
Comment (Grummel7) [inline default/scripting/species/SP_ABADDONI.focs.py:34]: This is again redundantly defined with the effects below, bound to produce inconsistencies in the future.
With python it should be possible define those standard effects and their associated tags from a single source.

Comment (o01eg) [reply default/scripting/species/SP_ABADDONI.focs.py:34]: I remember about it but not yet though how it should look.

**Comment by o01eg** [comment]
It's ready but I'm not sure if it should go to 0.5.0 release

**Review by Vezzra — APPROVED** [review]
Can build and run on macOS

**Comment by geoffthemedio** [comment]
> It's ready but I'm not sure if it should go to 0.5.0 release

what's special about these changes?

**Comment by o01eg** [comment]
@geoffthemedio Not so much but any changes made in species macroses should not be forgotten to be copied to appropriate Python modules. I expect species balance will happen before release and it could be easy mistake to make.

**Pull request merged**